introduction.md

```diff
@@ -1,3 +1,5 @@
-#Introduction
-Hello! My name is Paolo and I want to learn the secrets of Github.
-My favourite path is HTML & CSS
+* javascript
+* html
+* css
+* java
+* .net
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?